core/main/src/Utils/Utils.ts

```diff
@@ -224,47 +267,47 @@
~        return image;
~    }
~
~    public static deepExtend(destination: unknown, ...sources: unknown[]): unknown {
~        for (const source of sources) {
~            if (source === undefined || source === null) {
~                continue;
```

Comment:
Codacy found an issue: [The preceding `if` block ends with a `return` statement. This `else` is unnecessary.](https://app.codacy.com/gh/matteobruni/tsparticles/pullRequest?prid=6397531)